src/mmw/apps/modeling/models.py

```diff
@@ -78,6 +82,10 @@ class Project(models.Model):
     def __unicode__(self):
         return self.name
 
+    @property
+    def in_drb(self):
+        return self.area_of_interest.within(DRB)
```

Comment:
At first I added this as a method, but then it became apparent that having this easy check in the front-end would be useful, so I made it into a `@property` and included it in the serializer. Not sure what performance impact this may have to calculate this for every serialization request. Maybe we can find a different way to do it if it would be severe.